examples/ChatClient.py

```diff
@@ -4,7 +4,7 @@
 from time import sleep
```

Comment:
This way of doing import (along with the line below) is a horrible way to do imports (it's cluttering up the namespace)